research/schooltests/sql.txt

```diff
@@ -0,0 +1,126 @@
+
+Anweisung 2:
+	DELETE FROM Genre WHERE bezeichnung = "Klassik"
+
+	^(?:DELETE|ELETE|DLETE|DEETE|DELTE|DELEE|DELET)\s+(?:FROM|ROM|FOM|FRM|FRO)\s+(?:[`´'"]?(?:Genre|enre|Gnre|Gere|Gene|Genr)[`´'"]?(\s+(?:ALIAS|AS).*)?)\s+(?:WHERE|HERE|WERE|WHRE|WHEE|WHER)\s+(?:\s*\s*[`´'"]?(?:bezeichnung|ezeichnung|bzeichnung|beeichnung|bezichnung|bezechnung|bezeihnung|bezeicnung|bezeichung|bezeichnng|bezeichnug|bezeichnun)[`´'"]?\s*\s*=\s*\s*[`´'"]?(?:Klassik|lassik|Kassik|Klssik|Klasik|Klasik|Klassk|Klassi)[`´'"]?\s*\s*|\s*\s*[`´'"]?(?:Klassik|lassik|Kassik|Klssik|Klasik|Klasik|Klassk|Klassi)[`´'"]?\s*\s*=\s*\s*[`´'"]?(?:bezeichnung|ezeichnung|bzeichnung|beeichnung|bezichnung|bezechnung|bezeihnung|bezeicnung|bezeichung|bezeichnng|bezeichnug|bezeichnun)[`´'"]?\s*\s*);?$
+-------------------------------
+
+Leere alle Playlisten.
+
+DELETE FROM zugeordnetZu 
+
+^(?:DELETE|ELETE|DLETE|DEETE|DELTE|DELEE|DELET)\s+(?:FROM|ROM|FOM|FRM|FRO)\s+(?:[`´'"]?(?:zugeordnetZu|ugeordnetZu|zgeordnetZu|zueordnetZu|zugordnetZu|zugerdnetZu|zugeodnetZu|zugeornetZu|zugeordetZu|zugeordntZu|zugeordneZu|zugeordnetu|zugeordnetZ)[`´'"]?(\s+(?:ALIAS|AS).*)?);?$
+
+-------------------------------
+
+Erstelle eine neue Datenbank namens "Wetterdaten".
+
+CREATE DATABASE Wetterdaten
+
+(?:CREATE|REATE|CEATE|CRATE|CRETE|CREAE|CREAT)\s+(?:DATABASE|ATABASE|DTABASE|DAABASE|DATBASE|DATAASE|DATABSE|DATABAE|DATABAS)\s+((?:OR|R|O)\s+(?:REPLACE|EPLACE|RPLACE|RELACE|REPACE|REPLCE|REPLAE|REPLAC)\s+|(?:IF|F|I)\s+(?:NOT|OT|NT|NO)\s+(?:EXISTS|XISTS|EISTS|EXSTS|EXITS|EXISS|EXIST)\s+)?(?:Wetterdaten|etterdaten|Wtterdaten|Weterdaten|Weterdaten|Wettrdaten|Wettedaten|Wetteraten|Wetterdten|Wetterdaen|Wetterdatn|Wetterdate)
+
+-------------------------------
+
+Erstelle eine neue Tabelle "WetterStation" gemäß des folgenden Relationenmodells. 
+
+CREATE TABLE WetterStation (
+	id int NOT NULL,
+	name varchar(255),
+	ortId int NOT NULL,
+	PRIMARY KEY (id),
+    	FOREIGN KEY (ortId) REFERENCES orte(ortId)
```

Comment:
In the picture [img](https://github.com/StEx2022-23/sql2regex/pull/115/files)

it would be 

REFERENCES ort(orteid)

?

Reply:
thx - its done